dcat/index.html

```diff
@@ -14,7 +14,7 @@
 <!-- Disclaimer -->
 
 <section  id="abstract">
-    <p>DCAT is an RDF vocabulary designed to facilitate interoperability between data catalogs published on the Web.
+    <p>DCAT is an RDF vandal vocabulary designed to facilitate interoperability between data catalogs published on the Web.
```

Comment:
That's a bit tough. How about
```suggestion
    <p>DCAT is an RDF goth vocabulary designed to facilitate interoperability between data catalogs published on the Web.
```